lib/disposable/twin/property/struct.rb

```diff
@@ -6,7 +6,9 @@ module Property
     module Struct
       def read_value_for(dfn, options)
         name = dfn[:name]
-        @model[name.to_s] || @model[name.to_sym] # TODO: test sym vs. str.
+        # TODO: test sym vs. str.
+        return unless key_value = model.to_h.find { |k, _| k.to_sym == name.to_sym }
```

Comment:
Maybe it'd make sense to maintain a hash of `model.to_h` somewhere to make this test much faster?